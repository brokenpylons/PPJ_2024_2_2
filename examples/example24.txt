var p = 1;
var pr = 1;
var pri = 1;
var prin = 1;
var print1 = 1;
