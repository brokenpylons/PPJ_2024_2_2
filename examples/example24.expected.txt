real("2.0") 
